servers/rendering/renderer_rd/effects_rd.cpp

```diff
@@ -1914,7 +1914,7 @@ EffectsRD::EffectsRD(bool p_prefer_raster_effects) {
 		}
 	}
 
-	{ // Initialize copy
+	if (!prefer_raster_effects) { // Initialize copy
 		Vector<String> copy_modes;
```

Comment:
Nitpicking but seeing we disable this in raster it would be good to add a check at the start of the copy functions like I did with the other clustered functions I already disabled. This will allow us to spot if they are still used from somewhere even when the mobile renderer is used (or prevent someone from using the clustered ones without checking whether they are available).

Reply:
done